static/js/components/VegaPlot.jsx

```diff
@@ -56,7 +56,8 @@ const spec = (url) => ({
         },
         color: {
           field: "filter",
-          type: "nominal"
+          type: "nominal",
+          scale: { domain: ["ztfg", "ztfr", "ztfi"], range: ["#377E22", "#EA3323", "#CCCC52"] }
```

Comment:
I'm ok with an opinionated set of color assignments to different filters but shouldn't these all live in one place (e.g. the DB/enum file) and be grabbed dynamically from there? Also, what if the photometry is not from ztf (e.g. it's a Gatinni-only source)?

Reply:
Nice catch. This is an important thing to fix. We need to define the color that shows up for each filter, otherwise filters with undefined colors will not be plotted. Will push up a fix

Reply:
Just reverting to the Vega defaults for now (not specifying any specific colors for specific filters) which should fix the problem for the moment